rules/integrations/kubernetes/execution_user_exec_to_pod.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2022/05/17"
+maturity = "production"
+updated_date = "2022/05/17"
+integration = "kubernetes"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects a user attempt to establish a shell session into a pod using the 'exec' command. Using the 'exec' command in a pod allows a user to establish a temporary shell session and execute any process/commands in the pod. An adversary may call bash to gain a persistent interactive shell which will allow access to any data the pod has permissions to, including secrets. 
```

Comment:
toml-lint this please 🙏

Reply:
The jobs re-ran when I updated the False positives section, this should have toml-linted the file correct?

Reply:
No, you must use the python module (install instructions can be found here https://github.com/elastic/detection-rules/blob/main/README.md#getting-started), then use the `toml-lint` command

Reply:
Like: `py.exe -m detection_rules toml-lint -f .\rules\integrations\kubernetes\execution_user_exec_to_pod.toml`

Reply:
Thank you, I've toml-linted the file